nixos/doc/manual/release-notes/rl-unstable.xml

```diff
@@ -197,6 +197,33 @@ fileSystems."/example" = {
     <literal>services.hardware.opengl.extraPackages{,32}</literal> instead. You can
     also specify VDPAU drivers there.</para>
   </listitem>
+
+  <listitem>
+    <para>Input method support have been reorganized. Fcitx, ibus, nabi and uim are
+    now supported and new engines for fcitx (chewing, hangul, m17n, mozc and table-other)
+    and ibus (hangul and m17n) have been added.
+    Ibus has been moved to a new namespace
+    <literal>i18n.inputMethod.ibus</literal> and the <literal>plugins</literal> options
+    has been renamed to <literal>engines</literal> to standardize input methods options
+    names. The input method selection has been moved to a global
+    <literal>i18n.inputMethod.enabled</literal> option.
+    An example of the new style:
+
+<programlisting>
+i18n.inputMethod.enabled = "ibus";
+i18n.inputMethod.ibus.engines = with pkgs.ibus-engines; [ anthy mozc ];
+</programlisting>
+
+That is equivalent to the old version:
```

Comment:
There is no services for input method before, so I think just add them to the "new services" para should be enough.